recipe/meta.yaml

```
@@ -9,20 +9,17 @@ package:
 source:
   url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: 1205953abc24ff757f4a795304d5d61e4bf1e555c9ef6ec96a132d4b95535484
-  patches:
-    - setup.patch
-
 build:
-  number: 3
+  number: 0
+  # Skip Windows due to MSVC incompatibility with GCC extensions (__int128, packed structs, etc.)
+  skip: true  # [win or py<38]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation
 
 requirements:
   build:
-    - {{ compiler('c') }}  # [not win]
-    - {{ stdlib("c") }}    # [not win]
-    - {{ compiler('m2w64_c') }}  # [win]
-    - {{ stdlib("m2w64_c") }}    # [win]
+    - {{ compiler('c') }}
+    - {{ stdlib("c") }}
   host:
-    - libpython  # [win]
     - python
     - pip
     - setuptools
```

```suggestion
    - setuptools
    - poetry >=1.0.5
```

https://github.com/PetterS/quickjs/blob/1.19.4/pyproject.toml